umts-custom-cops.gemspec

```diff
@@ -22,16 +21,14 @@ Gem::Specification.new do |spec|
   end
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(spec)/}) }
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
```

Comment:
Why these changes? Just curious.

Reply:
Rubocop had some other opinions about the gemspec, and while I was cleaning up, I removed these two lines because they're not used.